target/linux/ath79/generic/base-files/etc/uci-defaults/04_led_migration

```diff
@@ -7,6 +7,11 @@ board=$(board_name)
 boardonly="${board##*,}"
 
 case "$board" in
+dlink,dap-1330-a1)
+	migrate_leds ":red:power=:red:status" \
+		":red:wifi=:red:rssilow" ":green:wifi=:green:rssimediumlow" \
+		":green:signal1=:green:rssimediumhigh" ":green:signal2=:green:rssihigh"
```

Comment:
Nitpick, but this might be nicer with one line per replacement:
```
	migrate_leds ":red:power=:red:status" \
		":red:wifi=:red:rssilow" \
		":green:wifi=:green:rssimediumlow" \
		":green:signal1=:green:rssimediumhigh" \
		":green:signal2=:green:rssihigh"
```